.github/workflows/visual_testing.yml

```diff
@@ -22,12 +22,12 @@ jobs:
           node-version: 16
 
       - name: Install dependencies
-        run: npm ci
+        run: npm ci && cd docs && npm ci && cd ..
 
       - name: Chromatic
         uses: chromaui/action@v1
         with:
-          buildScriptName: build:storybook
+          buildScriptName: ${{ (github.ref == 'refs/heads/main' && 'build:docs') || 'build:docs:preview' }}
```

Comment:
Would be faster if we don't have to build the docs first :)

We only need storybook to be deployed inside docs for deployment, not for visual testing :) so this could point to a raw `build-storybook` in package.json